src/include/dstore.h

```diff
@@ -108,9 +108,11 @@ int dstore_obj_write(struct dstore *dstore, void *ctx,
  * @paramp[in] obj - An open object.
  * @paramp[in] old_size - Current size of given object
  * @paramp[in] new_size - New size of a given object after performing this op
+ * @paramp[in] bsize - Block size of the associated file system
  * @return 0 on success -errno given by backend operation
  */
-int dstore_obj_resize(struct dstore_obj *obj, size_t old_size, size_t new_size);
+int dstore_obj_resize(struct dstore_obj *obj, size_t old_size, size_t new_size,
```

Comment:
Suggestion: Just like fs handle stores the block size, the block size should also be part of dstore object, since block size would be common across one dstore. The dstore_obj has pointer to dstore and the block size can be used/referenced from there rather than passing it here.
@sachinpunadikar @pratyush-seagate  thoughts?

Reply:
As per current logic, a FS will have fixed block size. So in principle every FS may have different block size.
In that context, whether dstore_obj has one-to-one mapping with FS ? If so, then bsize can be part of dstore_obj, else not.

Reply:
Currently the file system block size and dstore block size (internally motr block size) are same. Is it possible that in future we may need to support file system with different block size than dstore? If so, in that case we should avoid storing fs block size in dstore.

Another point is, in the file system stack, block size is usually saved in the stat attribute st_blksize (for files, directories and other FS objects, they inherit it from parent file system). If dstore is not aware about other file system specific attributes like no of blocks etc., then IMHO it doesn't need the fs block size to be stored implicitly. The callers can always pass this parameter as required.

Reply:
Ack. Resolving this comment.